frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.scss

```diff
@@ -44,3 +44,11 @@
     margin-bottom: var(--pf-global--spacer--md);
   }
 }
+
+.ocs-storageClass-encryption__form-radio {
+  margin-bottom: var(--pf-global--spacer--md) !important;
+}
+
+.ocs-storageClass-encryption__form--padding {
+  padding-left: var(--pf-global--spacer--md) !important;
+}
```

Comment:
You can move this inside:
ocs-storage-class-encryption {
   &__form-radio{
  }
 &__form--padding{
}
}

ocs-storage-class-encryption is already there